#request

---
#response
geometry_msgs/PoseStamped current_pose
